pkgs/by-name/ec/ecm/package.nix

```diff
@@ -4,22 +4,29 @@
   fetchurl,
   gmp,
   m4,
+  coreutils,
```

Comment:
Was this used?

Reply:
No, please remove it.

Reply:
Oops.  At one point I had `${coreutils}/bin/rm`, but I found out I just needed `rm`.  Will fix.